test/schema_plus/core_test.clj

```diff
@@ -196,12 +195,13 @@
 (declare +MyPerson-with-name)
 (declare +MyPerson-with-age)
 (declare +MyPerson-build)
+(declare +MyPerson->)
+(defschema+ MyPerson
```

Comment:
there is something weird going on here. if MyPerson isn't def'd at the top level the generated macro isn't quite right (it becomes a function somehow). for now I'm just punting on figuring out why

Reply:
Everything besides the convenience macro works in that case, though?

Reply:
yeah the tests without that macro passed